src/ChickenAPI.Packets.Serialization/Deserializers/World/Chat/ClientSayPacketDeserializer.cs

```diff
@@ -0,0 +1,18 @@
+﻿using ChickenAPI.Packets.Game.Client.Chat;
+
+namespace ChickenAPI.Packets.Serialization.Deserializers.World.Chat
+{
+    public class ClientSayPacketDeserializer : GenericBasePacketDeserializer<ClientSayPacket>
+    {
+        protected override string Header => "say";
```

Comment:
Would suggest to use a property here
![SharpLab](https://user-images.githubusercontent.com/35202750/54666959-14cda880-4b50-11e9-9659-5d7d11d26557.png)

Reply:
Thanks, could you tell the name of the tool are you using to see the IL that fast ?

Reply:
https://sharplab.io

Reply:
![image](https://user-images.githubusercontent.com/16936368/54668804-be01a800-4b2a-11e9-8539-8d5262da5764.png)
Actually, a field can not be defined as `abstract`